src/components/Header.tsx

```diff
@@ -12,17 +12,38 @@ import { Link } from "react-router-dom";
 import { useAuthState, useAuthDispatch, logout } from "../contexts";
 import { routes } from "../constants";
 import logo from "../static/logo.png";
-import { Role } from "../types";
+import Joyride, { CallBackProps, STATUS, StoreHelpers } from 'react-joyride';
+import { HelpOutline } from "@material-ui/icons";
 
-const Header: FunctionComponent = () => {
+const Header: FunctionComponent = (props: any) => {
   const [menuRef, setMenuRef] = React.useState<null | HTMLElement>(null);
+  const [run, setRun] = React.useState(false);
   const { loggedIn, user } = useAuthState();
   const authDispatch = useAuthDispatch();
+  let helpers: StoreHelpers;
```

Comment:
this one is not used 
I checked it works without 
do we need it?

Reply:
Removed.